index.html

```diff
@@ -27,15 +43,39 @@ <h5>Mike Mikerson</h5>
               <span class="title-bar-title">Mike Mikerson</span>
             </div>
           </div>
+
+          <!-- Search Bar -->
           <div class="callout primary">
             <div class="row column">
-              <h1>Hello! This is the portfolio of a very witty person.</h1>
-              <p class="lead">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In condimentum facilisis porta. Sed nec diam eu diam mattis viverra. Nulla fringilla.</p>
+              <h1>Search for Investors</h1>
+              <!-- <p class="lead">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In condimentum facilisis porta. Sed nec diam eu diam mattis viverra. Nulla fringilla.</p> -->
+              <div class="input-group input-group-rounded">
+                <input class="input-group-field" type="search">
+                <div class="input-group-button">
+                  <input type="submit" class="button secondary" value="Search">
+                </div>
+              </div>
+              
+              </div>
+              
+                
+              
             </div>
-          </div>
```

Comment:
Hey Jaden! Can you check to see if this div closure removal was necessary or not?

Reply:
YES IT WAS